From 2eae3094d018587e2550aaf895a7cbdeaea679bd Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Fri, 20 Jan 2023 18:40:57 +0100
Subject: [PATCH] bootctl: remove space that should not be there

(cherry picked from commit e684d2d5f85a82ed47eb063809145540df01ae1a)

Related: RHEL-16354
---
 src/boot/bootctl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/boot/bootctl.c b/src/boot/bootctl.c
index 4f2a6288fb..3044f67b5a 100644
--- a/src/boot/bootctl.c
+++ b/src/boot/bootctl.c
@@ -1433,7 +1433,7 @@ static int help(int argc, char *argv[], void *userdata) {
         if (r < 0)
                 return log_oom();
 
-        printf("%1$s  [OPTIONS...] COMMAND ...\n"
+        printf("%1$s [OPTIONS...] COMMAND ...\n"
                "\n%5$sControl EFI firmware boot settings and manage boot loader.%6$s\n"
                "\n%3$sGeneric EFI Firmware/Boot Loader Commands:%4$s\n"
                "  status              Show status of installed boot loader and EFI variables\n"
